config_schema.json

```diff
@@ -3,6 +3,54 @@
   "description": "Modifies the orginal Settings class provided by the user",
   "type": "object",
   "properties": {
+    "files_to_register_success_type": {
+      "title": "Files To Register Success Type",
+      "description": "The type used for events informing about the success of a file validation.",
+      "default": "file_validation_success",
+      "example": "file_validation_success",
+      "env_names": [
+        "irs_files_to_register_success_type"
+      ],
+      "type": "string"
+    },
+    "files_to_register_failure_type": {
+      "title": "Files To Register Failure Type",
+      "description": "The type used for events informing about the failure of a file validation.",
+      "default": "file_validation_failure",
+      "example": "file_validation_failure",
+      "env_names": [
+        "irs_files_to_register_failure_type"
+      ],
+      "type": "string"
+    },
+    "files_to_register_topic": {
```

Comment:
It might be better to define these topics and events from the perspective of the Interrogation room service a d for instance call the topic "interrogations". However, this is not really critical.